openslides/utils/settings.py.tpl

```diff
@@ -112,6 +112,11 @@ if use_redis:
     SESSION_ENGINE = 'redis_sessions.session'
 
 
+# When use_redis is True, the restricted data cache caches the data individuel
+# for each user. This requires a lot of memory if there are a lot of active
+# users. If use_redis is False, this setting has no effect.
+USE_RESTRICTED_DATA_CACHE = True
```

Comment:
Let us please use a falsy default: `DISABLE_USER_CACHE=False` Also we have to explain in three words what the restricted data cache is. Non developers won't know.

Reply:
Done.